rules_building_block/collection_linux_suspicious_clipboard_activity.toml

```diff
@@ -0,0 +1,52 @@
+[metadata]
+creation_date = "2023/07/27"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "Multiple field support in the New Terms rule type was added in Elastic 8.6"
+min_stack_version = "8.6.0"
+updated_date = "2023/07/27"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule monitors for the usage of the most common clipboard utilities on unix systems by an uncommon process group 
+leader. Adversaries may collect data stored in the clipboard from users copying information within or between 
+applications. 
+"""
+from = "now-119m"
+interval = "60m"
+index = ["logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "Potential Suspicious Clipboard Activity Detected"
+risk_score = 21
+rule_id = "884e87cc-c67b-4c90-a4ed-e1e24a940c82"
+severity = "low"
+tags = ["Domain: Endpoint", "OS: Linux", "Use Case: Threat Detection", "Tactic: Collection", "Rule Type: BBR"]
+type = "new_terms"
+building_block_type = "default"
+query = '''
+event.category:process and host.os.type:"linux" and event.action:"exec" and event.type:"start" and 
+process.name:("xclip" or "xsel" or "wl-clipboard" or "clipman" or "copyq")
```

Comment:
Just checking, should we consider it suspicious for a user to use the clipboard manager intended to be used with their desktop environment? I may just be missing where in the query it restricts the context to `an uncommon process group leader`. But, the way I read this rule, it could throw false positives whenever the clipboard manager starts, which would often be on boot. Thanks! 

Context in case it is useful:
Typically with Linux there are two clipboard often present, the in-built one to Linux and then the one that is designed for your desktop environment. However, these are not always installed in the base image of the OS. `wl-clipboard` for instance is what is paired with Wayland (which most Ubuntu users should be using), and `xclip` is paired with KDE, `clipman` for XFCE, etc..